composer.json

```diff
@@ -18,15 +18,14 @@
     "config": {
         "sort-packages": true
     },
-    "extra": {
-    },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.3 || ^8.0",
```

Comment:
Unfortunately, PHP does not follow semantic versioning therefore we use `~8.0.0` and `~8.1.0`.

Reply:
was by purpose to prevent another new version for this package on the next minor php version. but you're right... there's no guarantee things will not break in the next minor...

Reply:
> was by purpose to prevent another new version for this package on the next minor php version.

Good idea, but unfortunately it doesn't work. 😞 

For version 8.1 see: https://www.php.net/releases/8.1/en.php#deprecations_and_bc_breaks